cmd/nerdctl/image_convert.go

```diff
@@ -67,6 +68,16 @@ func newImageConvertCommand() *cobra.Command {
 	imageConvertCommand.Flags().Bool("zstdchunked", false, "Use zstd compression instead of gzip (a.k.a zstd:chunked). Should be used in conjunction with '--oci'")
 	// #endregion
 
+	// #region nydus flags
+	imageConvertCommand.Flags().Bool("nydus", false, "Convert an OCI image to Nydus image. Should be used in conjunction with '--oci'")
```

Comment:
Needs docs about usage 

```
# nerdctl image convert --nydus --oci ubuntu:22.04 ubuntu:22.04-nydus
ERRO[0000] fail to run nydus-image [create --log-level warn --prefetch-policy fs --blob /tmp/nydus-converter-1770408557/blob --source-type directory --whiteout-spec none --fs-version 5 --inline-bootstrap --compressor lz4_block /tmp/nydus-converter-1770408557/source]  error="exec: \"nydus-image\": executable file not found in $PATH"
FATA[0000] copy nydus blob to content store: failed to copy: pack nydus tar: reading from a closed fifo 
```

Reply:
Thanks for the review! I can add some usage to [nydus.md](https://github.com/containerd/nerdctl/blob/master/docs/nydus.md).

Reply:
Is this addressed?

Reply:
@ktock Yes, I updated the doc just now.